tools/accuracy_checker/configs/resnet-50-cf2.yml

```diff
@@ -0,0 +1,46 @@
+models:
+
+  - name: resnet-50-cf2
+    launchers:
+      - framework: onnx_runtime
+        model: public/resnet-50-cf2/resnet-50-cf2.onnx
+        adapter: classification
+
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+        - type: resize
+          size: 256
+          aspect_ratio_scale: greater
+        - type: crop
+          size: 224
+        - type: normalization
+          mean: 103.53, 116.28, 123.675
+          std: 57.375, 57.12, 58.395
+
+  - name: resnet-50-cf2
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model: public/resnet-50-cf2/FP32/resnet-50-cf2.xml
```

Comment:
please align model path start with weight path start for consistency with other AC configs. It is applicable for all added AC configs.